config/300-repositories.yaml

```diff
@@ -170,7 +170,11 @@ spec:
                         name:
                           type: string
                           description: "The secret name"
-
+                repo_list_to_scope_token:
+                  type: array
+                  items:
+                    description: list of repoitories where GitHub token can be scoped
```

Comment:
the official word is Github

Reply:
But I see everywhere in github docs as `GitHub` 

How do we know which one is official :thinking:

Reply:
ah yeah my mistake 😅